### DISCUSSION TITLE
Can't find Task package in lit-all bundle

### DISCUSSION THREAD

**Discussion by shmox75** [lit/lit#4620] [Q&A] [answered]
Hi I'm developing without build tools and without typescript. I want to add lit to my existing website.
All seems ok until I tried to import Task pacakage, it throws error:
`language-selector.js:2 Uncaught SyntaxError: The requested module './lit-all.min.js' does not provide an export named 'Task'`

Here is the import code:

```
import { LitElement, html, css, Task } from './lit-all.min.js';

export class LanguageSelector extends {
............
}
```

**Comment by justinfagnani** [comment, marked as answer]
The bundles only include the core libraries of lit-html, @lit/reactive-element, and lit-element.

We publish too many other optional libraries to include them in the bundles. If you need them you should use npm or a CDN like jsdelivr.

**Reply by shmox75** [reply]
Many thanks. I wonder if there is a way to download & use a jsdelivr file locally ? Tried it but it fails with error:
`Failed to load module script: Expected a JavaScript module script but the server responded with a MIME type of "text/html". Strict MIME type checking is enforced for module scripts per HTML spec.`

**Reply by justinfagnani** [reply]
If you want to download packages locally, you should use npm. That's exactly what it's for.

**Reply by shmox75** [reply]
Thanks finally got it working. But now I lost styling. Very strange.
Edit: Nevermind it was an error in my code.
I can't thank you enough for your patience and your help. <3
